infobip-typescript-generator-extension-annotation-processor-showcase/dist/CommonValidationMessages.ts

```diff
@@ -1,4 +1,5 @@
-const localize = (message, object: object = {}) => format(message, object);
+import { localize } from './Localization';
```

Comment:
jel mozes malo pojasniti kako ovaj localization radi?
takodjer vidim da u toj metodi localize vraca void, a pretpostavljam da bi trebao vratiti string, jel to greska

Reply:
totalno si u pravu to ne radi ništa drugo, nego samo mijenja placeholdere ajde ja revisitam to jer ima i bug pa vidim da li možemo to renamati u skolopu bug fixa tako da ubijemo dvije muhe jednim udarcem :)

Reply:
popravljena stvar s voidom i podržana zamjena generiranja placeholdera s naše strane, još je ostao rename

Reply:
nećemo renamti, napravio sam istraživanje, ovo je Lokaliyacija pa ostacim as is, a u doksi objasnim kako se to treba koristiti